src/tests/ftest/util/fio_utils.py

```diff
@@ -194,236 +194,246 @@ def _run_process(self):
+            self.stats = FormattedParameter("--stats={}")
+            self.zero_buffers = FormattedParameter("--zero_buffers={}")
+            self.refill_buffers = FormattedParameter("--refill_buffers={}")
+            self.scramble_buffers = FormattedParameter("--scramble_buffers={}")
+            self.buffer_pattern = FormattedParameter("--buffer_pattern={}")
+            self.buffer_compress_percentage = FormattedParameter(
+                "--buffer_compress_percentage={}")
+            self.buffer_compress_chunk = FormattedParameter(
+                "--buffer_compress_chunk={}")
+            self.dedupe_percentage = FormattedParameter(
+                "--dedupe_percentage={}")
+            self.clat_percentiles = FormattedParameter("--clat_percentiles={}")
+            self.lat_percentiles = FormattedParameter("--lat_percentiles={}")
+            self.percentile_list = FormattedParameter("--percentile_list={}")
+            self.significant_figures = FormattedParameter(
+                "--significant_figures={}")
+            self.disk_util = FormattedParameter("--disk_util={}")
+            self.gtod_reduce = FormattedParameter("--gtod_reduce={}")
+            self.disable_lat = FormattedParameter("--disable_lat={}")
+            self.disable_clat = FormattedParameter("--disable_clat={}")
+            self.disable_slat = FormattedParameter("--disable_slat={}")
+            self.disable_bw_measurement = FormattedParameter(
+                "--disable_bw_measurement={}")
+            self.gtod_cpu = FormattedParameter("--gtod_cpu={}")
+            self.unified_rw_reporting = FormattedParameter(
+                "--unified_rw_reporting={}")
+            self.continue_on_error = FormattedParameter(
+                "--continue_on_error={}")
+            self.error_dump = FormattedParameter("--error_dump={}")
+            self.profile = FormattedParameter("--profile={}")
+            self.cgroup = FormattedParameter("--cgroup={}")
+            self.cgroup_nodelete = FormattedParameter("--cgroup_nodelete={}")
+            self.cgroup_weight = FormattedParameter("--cgroup_weight={}")
+            self.uid = FormattedParameter("--uid={}")
+            self.gid = FormattedParameter("--gid={}")
+            self.kb_base = FormattedParameter("--kb_base={}")
+            self.unit_base = FormattedParameter("--unit_base={}")
+            self.hugepage_size = FormattedParameter("--hugepage-size={}")
+            self.flow_id = FormattedParameter("--flow_id={}")
+            self.flow = FormattedParameter("--flow={}")
+            self.flow_watermark = FormattedParameter("--flow_watermark={}")
+            self.flow_sleep = FormattedParameter("--flow_sleep={}")
+            self.steadystate = FormattedParameter("--steadystate={}")
+            self.steadystate_duration = FormattedParameter(
+                "--steadystate_duration={}")
+            self.steadystate_ramp_time = FormattedParameter(
+                "--steadystate_ramp_time={}")
```

Comment:
Was that somehow automatically generated?

Reply:
I'm not sure I understand the question.  The only change in this file is moving the `FioJob` class to now be a member of the `FioCommand` class.  The optional command line parameters did not change.